### DISCUSSION TITLE
2 PR's merged but still didn't got badge!!

### DISCUSSION THREAD

**Discussion by RajeevRanjan5642** [community/community#131066] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Below is the screenshot of my merged PR's : 
![Screenshot 2024-07-05 075519](https://github.com/community/community/assets/95275538/f5e4a89b-70e6-436f-bada-15351afe6806)
Do I need to do something else in order to get the badge??

**Comment by asdacosta** [comment]
If you unlocked the achievement with private contributions, you can only show them off by including private contributions in your Profile.

**Reply by RajeevRanjan5642** [reply]
Sorry I didn't get you. These are are the public repos to which I have contributed. Will u be more elaborative?

**Reply by asdacosta** [reply, marked as answer]
Turn on each section in Contribution Settings, just above your contribution graph. That will make sure your badges are not hidden.

**Comment by RajeevRanjan5642** [comment]
Thank u so much 💓.
It's showing now.

**Reply by asdacosta** [reply]
Good to hear it’s showing @RajeevRanjan5642. 
Kindly mark my specific response that answered your question as the answer to this discussion to help others with similar problem in the future.

**Reply by RajeevRanjan5642** [reply]
Done ✔️ 👍🏽
